### PR TITLE
New logging for propers.py

### PR DESCRIPTION
**Pull request by fernandog**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/search/proper.py [hunk 1]
```diff
@@ -123,61 +113,57 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
...
                 cur_propers = cur_provider.find_propers(recently_aired)
             except AuthException as e:
-                logger.log('Authentication error: {error}'.format
-                           (error=ex(e)), logger.DEBUG)
+                log.debug('Authentication error: {error}', {'error': ex(e)})
                 continue
             except socket_timeout as e:
-                logger.log('Socket time out while searching for propers in {provider}, skipping: {error}'.format
-                           (provider=cur_provider.name, error=ex(e)), logger.DEBUG)
+                log.debug('Socket time out while searching for propers in {provider}, skipping: {error}',
```
Comment (p0psicles) [inline medusa/search/proper.py]: Can you remove the ex import and these function calls. And remove them with {error!r}

File: medusa/search/proper.py [hunk 2]
```diff
@@ -123,61 +113,57 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
...
+            log.info('Searching for any new PROPER releases from {provider}', {'provider': cur_provider.name})
 
             try:
                 cur_propers = cur_provider.find_propers(recently_aired)
             except AuthException as e:
-                logger.log('Authentication error: {error}'.format
-                           (error=ex(e)), logger.DEBUG)
+                log.debug('Authentication error: {error}', {'error': ex(e)})
                 continue
             except socket_timeout as e:
```
Comment (p0psicles) [inline medusa/search/proper.py]: We've started renaming e to error. As 'e'  is rather meaningless.

File: medusa/search/proper.py [hunk 3]
```diff
@@ -196,38 +182,40 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
...
                 continue
 
             if not cur_proper.parse_result.proper_tags:
-                logger.log('Skipping non-proper: {name}'.format(name=cur_proper.name))
+                log.info('Skipping non-proper: {name}', {'name': cur_proper.name})
                 continue
 
-            logger.log('Proper tags for {proper}: {tags}'.format
-                       (proper=cur_proper.name, tags=cur_proper.parse_result.proper_tags), logger.DEBUG)
+            log.debug('Proper tags for {proper}: {tags}',
```
Comment (p0psicles) [inline medusa/search/proper.py]: Can you use the same alignment as in your previous Pr?

File: medusa/search/proper.py [hunk 4]
```diff
@@ -270,14 +256,14 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
                                              b"showid = ? AND season = ? AND episode = ? AND status LIKE '%04'",
                                              [best_result.indexerid, best_result.actual_season, best_result.actual_episodes[0]])
             if not sql_results:
-                logger.log("Ignoring proper because this episode doesn't have 'DOWNLOADED' status: {name}".format
-                           (name=best_result.name))
+                log.info("Ignoring proper because this episode doesn't have 'DOWNLOADED' status: {name}".format
```
Comment (p0psicles) [inline medusa/search/proper.py]: You've missed this one.

File: medusa/search/proper.py [hunk 5]
```diff
@@ -287,14 +273,15 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
...
+                if all([current_codec, best_result.parse_result.video_codec,
+                        best_result.parse_result.video_codec != current_codec]):
+                    log.info('Ignoring proper because codec is different: {name}', {'name': best_result.name})
                     if best_result.name not in processed_propers_names:
                         self.processed_propers.append({'name': best_result.name, 'date': best_result.date})
                     continue
             else:
-                logger.log("Coudn't find a release name in database. Skipping codec comparison for: {name}".format
-                           (name=best_result.name), logger.DEBUG)
+                log.debug("Coudn't find a release name in database. Skipping codec comparison for: {name}".format
```
Comment (p0psicles) [inline medusa/search/proper.py]: Missed

**Comment by fernandog** [comment]
@p0psicles removed all exception handler from propers.py as MedusaSafeSession already handle it. Ok?

**Pull request merged**